docs/diagrams/ArchitectureSequenceDiagram.puml

```diff
@@ -19,7 +19,7 @@ activate model MODEL_COLOR
 model -[MODEL_COLOR]-> logic
 deactivate model
 
-logic -[LOGIC_COLOR]> storage : saveAddressBook(addressBook)
+logic -[LOGIC_COLOR]> storage : saveAddressBook(patientist)
```

Comment:
Not very sure if we even need to edit this file actually but I think this should be savePatientist?